assets/js/theme/global/compare-products.js

```diff
@@ -57,7 +58,10 @@ export default function (urlContext) {
         const productsToCompare = $this.find('input[name="products\[\]"]:checked');
 
         if (productsToCompare.length <= 1) {
-            alert('You must select at least two products to compare');
+            swal({
+                text: 'You must select at least two products to compare',
```

Comment:
This is an existing problem. So you probably don't need to fix it in this PR. Just pointing it out that the warning message needs to come from the language file, not hard-coded in here.